src/OpenColorIO/OCIOYaml.cpp

```diff
@@ -4,6 +4,7 @@
 #include <cstring>
 
 #include <OpenColorIO/OpenColorIO.h>
+#include <unordered_set>
```

Comment:
We order all the includes, and also regroup them based on 'family'. So, you should move that one up to line 5 (i.e. keep the empty line with the existing include at line 6).

Reply:
To clarify the point:
```
#include <cstring>
#include <unordered_set>

#include <OpenColorIO/OpenColorIO.h>
```